modules/openapi-generator/src/main/java/org/openapitools/codegen/CodegenSecurity.java

```diff
@@ -25,9 +25,10 @@
 
 public class CodegenSecurity {
     public String name;
+    public String description;
     public String type;
     public String scheme;
-    public Boolean isBasic, isOAuth, isApiKey;
+    public Boolean isBasic, isOAuth, isApiKey, isOpenId;
```

Comment:
FYI @OpenAPITools/generator-core-team 

This PR also adds supports for openID security definition.